README.md

```diff
@@ -5,30 +5,33 @@
 ./gradlew build
 ```
 
-## Running
-Boot up an [Apache Cassandra](https://cassandra.apache.org/) instance
-using the default hostname and ports: `localhost:9042`
-```
-./gradlew run
-```
-## Build Checks
-* ### OWASP check
-	`./gradlew dependencyCheckAnalyze --info`
+### Running Tests
+Tests are run automatically with `./gradlew build`. The tests require that Docker is running. Even if the tests fail, the artifacts are built and can be run.
```

Comment:
✏️ Integration tests require...

Reply:
Thanks. I just confirmed that `./gradlew clean build -PskipITests` succeeds even if Docker isn't running.